doc/source/api.rst

```diff
@@ -449,109 +449,103 @@ Datetimelike Properties
 
 ``Series.dt`` can be used to access the values of the series as
 datetimelike and return several properties.
-Due to implementation details the methods show up here as methods of the
-``DatetimeProperties/PeriodProperties/TimedeltaProperties`` classes. These can be accessed like ``Series.dt.<property>``.
```

Comment:
yes, I know `s.str?` works, and there the `StringMethods` name is visible to users, and it was for this reason I was thinking if we should mention it or not